### PR TITLE
1740 User context from token endpoint

### PR DESCRIPTION
**Pull request by Daniel-Echegaray**

### CODE REVIEW

File: Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift [hunk 1]
```diff
@@ -17,6 +19,23 @@ public struct ClientConfiguration {
...
+                sessionServiceStr = "https://session-service.login.schibsted.com"
+            case .proFi:
+                sessionServiceStr = "https://session-service.login.schibsted.fi"
+            case .proNo:
+                sessionServiceStr = "https://session-service.payment.schibsted.no"
+            case .pre:
+                sessionServiceStr = "https://session-service.identity-pre.schibsted.com"
+            case .proDk:
+                sessionServiceStr = "https://session-service.login.schibsted.dk"
+            }
```
Comment (Daniel-Echegaray) [inline Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift:36]: @saltwoo is this the right urls for the different environment of session-service?

File: Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift [hunk 2]
```diff
@@ -29,12 +48,14 @@ public struct ClientConfiguration {
     
     public init(environment: Environment, clientId: String, redirectURI: URL) {
         self.init(serverURL: URL(string: environment.rawValue)!,
+                  sessionServiceURL: URL(string: environment.sessionService)!,
                   clientId: clientId,
                   redirectURI: redirectURI)
     }
     
-    public init(serverURL: URL, clientId: String, redirectURI: URL) {
+    init(serverURL: URL, sessionServiceURL: URL, clientId: String, redirectURI: URL) {
```
Comment (Daniel-Echegaray) [inline Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift:56]: This one probably could be unpublic @zamorastp

Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift:56]: Are there some scenarios where they would like to use this method?  Test against another server?

**Review by zamorastp — APPROVED** [review]
Looks good for me :) just look at my question as answer to your question :D

File: Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift (hunk 1 above)
Comment (saltwoo) [reply Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift:36]: yes, that is correct!

File: Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift (hunk 2 above)
Comment (Daniel-Echegaray) [reply Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift:56]: Perhaps. Making public again

Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift]: If we dont know, lets go and make it private. Someone will ask if they have used it ;)

Comment (Daniel-Echegaray) [reply Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift]: Back to internal then ;). This will be a breaking API change then. FYI

**Pull request merged**